Cannot copy some `Text`s

This is quite an interesting problem. Trying to copy a workspace (or only its text morph for that matter) will shatter the renderer completely, resulting in a frozen world which needs to be restarted.
However, a browser **as a whole** can be copied. Only its text will crash again.

Both of these cases result in the following:

```js
renderer.js:237 Error rendering morphs: TypeError: Cannot read properties of null (reading 'style')
    at Window.stylepropsToNode (property-dom-mapping.js:13:12)
    at Renderer.Renderer_patchTextLayerStyleObject_ (renderer.js:1657:31)
    at Text.Text_patchSpecialProps_ (morph.js:2213:26)
    at Renderer.Renderer_renderStylingChanges_ (renderer.js:557:23)
    at Renderer.Renderer_renderStep_ (renderer.js:283:22)
    at eval (renderer.js:235:24)
```

A vanilla text morph can be copied.
Copying an editable vanilla text results in 

```js
renderer.js:237 Error rendering morphs: TypeError: Cannot read properties of null (reading 'classList')
    at Renderer.Renderer_patchLineWrapping_ (renderer.js:1671:25)
    at Text.Text_patchSpecialProps_ (morph.js:2254:28)
    at Renderer.Renderer_renderStylingChanges_ (renderer.js:557:23)
    at Renderer.Renderer_renderStep_ (renderer.js:283:22)
    at eval (renderer.js:235:24)
```

--- a/lively.morphic/morph.js
+++ b/lively.morphic/morph.js
@@ -896,7 +896,7 @@ export class Morph {
     this._env = MorphicEnv.default(); // FIXME!
     this._rev = snapshot.rev;
     this._owner = null;
-    this._id = objRef.id;
+    this._id = pool.reinitializeIds ? pool.reinitializeIds(objRef.id, objRef) : objRef.id;
     this._animationQueue = new AnimationQueue(this);
     this._cachedPaths = {};
     this._pathDependants = [];
@@ -909,7 +909,9 @@ export class Morph {
     const s = pool.expressionSerializer;
     for (const prop in this._parametrizedProps) {
       const v = this._parametrizedProps[prop].value;
-      if (v && obj.isString(v) && s.isSerializedExpression(v)) { this._parametrizedProps[prop] = s.deserializeExpr(v); }
+      if (v && obj.isString(v) && s.isSerializedExpression(v)) {
+        this._parametrizedProps[prop] = s.deserializeExpr(v);
+      }
     }
     this.initializeProperties();
     this._pool = pool;

--- a/lively.serializer2/object-pool.js
+++ b/lively.serializer2/object-pool.js
@@ -246,7 +246,8 @@ export class ObjectPool {
     // add more / custom stuff to snapshot or modify it somehow
     const addFn = this.plugins.additionallySerialize.length
       ? __additionally_serialize__addObjectFunction(
-          ref, snapshot, serializedObjMap, this, path) : null;
+        ref, snapshot, serializedObjMap, this, path)
+      : null;
     for (let i = 0; i < this.plugins.additionallySerialize.length; i++) {
       const p = this.plugins.additionallySerialize[i];
       p.additionallySerialize(this, ref, snapshot, addFn);
@@ -288,7 +289,8 @@ function __additionally_serialize__addObjectFunction (objRef, snapshot, serializ
   // other objects to serialization
 
   return (key, value, verbatim = false) =>
-    snapshot.props[key] = verbatim ? { value, verbatim }
+    snapshot.props[key] = verbatim
+      ? { value, verbatim }
       : {
           value: objRef.snapshotProperty(
             objRef.id, value, path.concat([key]), serializedObjMap, pool)
@@ -348,7 +350,8 @@ export class ObjectRef {
 
   get currentRev () {
     return this.snapshotVersions.length
-      ? arr.last(this.snapshotVersions) : 0;
+      ? arr.last(this.snapshotVersions)
+      : 0;
   }
 
   asRefForSerializedObjMap (rev = '????') {
@@ -486,7 +489,12 @@ export class ObjectRef {
     }
 
     const idPropertyName = newObj.__serialization_id_property__ || this.idPropertyName;
-    if (pool.reinitializeIds && newObj.hasOwnProperty(idPropertyName)) { newObj[idPropertyName] = pool.reinitializeIds(this.id, this); }
+    const needsReinitialization = newObj[idPropertyName] === this.id;
+    if (pool.reinitializeIds &&
+        newObj.hasOwnProperty(idPropertyName) &&
+        needsReinitialization) {
+      newObj[idPropertyName] = pool.reinitializeIds(this.id, this);
+    }
 
     pool.plugin_additionallyDeserializeAfterProperties(
       this, newObj, snapshot, serializedObjMap, path);
@@ -497,7 +505,8 @@ export class ObjectRef {
   recreatePropertyAndSetProperty (newObj, props, key, serializedObjMap, pool, path) {
     try {
       var { verbatim, value } = props[key] || { value: 'NON EXISTING' };
-      newObj[key] = verbatim ? value
+      newObj[key] = verbatim
+        ? value
         : this.recreateProperty(key, value, serializedObjMap, pool, path.concat(key));
     } catch (e) {
       let objString;

